src/tools/gcc.lua

```diff
@@ -96,6 +95,10 @@
 		},
 		symbols = {
 			On = "-g"
+		},
+		omitframepointer = {
```

Comment:
Does Clang automatically pull this in? I feel like it doesn't, but it definitely should since the flag was pulled in.